File: php/testcode/benchmark_test_00124.php
<?php
require_once __DIR__ . '/shared.php';

function benchmarkTest00124(BenchmarkRequest $req): BenchmarkResponse {
    session_set_cookie_params(['httponly' => true, 'samesite' => 'Lax']);
    session_start();
    $_SESSION['user'] = $req->post('username');
    return BenchmarkResponse::ok('Session started');
}

File: rust/testcode/benchmark_test_00124.rs
pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let username = req.param("username");
    if username.is_empty() {
        return super::shared::BenchmarkResponse::bad_request("Required");
    }
    let result = create_account(&username);
    super::shared::BenchmarkResponse::ok(&result)
}

fn create_account(name: &str) -> String { format!("created={}", name) }
